__e2e__/peruse.spec.js

```diff
@@ -340,6 +340,52 @@ describe( 'main window', () =>
         expect( header ).toBe( 'History' );
     } );
 
+    it( 'checks for Mock Network Tag', async () =>
+    {
+        expect.assertions( 1 );
+        const { client } = app;
+        await delay( 4500 );
+        await newTab( app );
+        await delay( 2500 );
+        if ( nodeEnv === 'dev')
+        {
+            expect( await client.waitForExist( BROWSER_UI.MOCK_TAG, WAIT_FOR_EXIST_TIMEOUT ) ).toBeTruthy();
+        }
+        if ( nodeEnv === 'prod')
+        {
+            expect( await client.isExisting( BROWSER_UI.MOCK_TAG, WAIT_FOR_EXIST_TIMEOUT, true ) ).toBeFalsy();
```

Comment:
Just running this text, in mock passes. Ie: this test doesn't appear to actually check what you want...

Why is it different to the above test? (ln 352?)

Reply:
These should really be two separate tests (running only if in that environment)